client/js/widgets/widget.js

```diff
@@ -257,12 +260,18 @@ export class Widget extends StateManaged {
       className += ' foreign';
     if(typeof this.get('owner') == 'string' && this.get('owner') != playerName)
       className += ' foreign';
+    if(this.get('onlyVisibleForSeat'))
+      if(!widgetFilter(w=>w.get('player') == playerName && toArray(this.get('onlyVisibleForSeat')).indexOf(w.get('id')) != -1).length)
```

Comment:
Perhaps check here for `w.get('type') == 'seat'`, as you do in the next `if`?

Reply:
Yeah, not sure what you mean. `onlyVisibleForSeat` and `linkedToSeat` are meant to link _other_ widgets to a seat.